refinery/selenium_testing/tests.py

```diff
@@ -26,6 +29,10 @@ class SeleniumTestBase(StaticLiveServerTestCase):
 
     def setUp(self, site_login=True, initialize_guest=True,
               public_group_needed=True):
+
+        # Start a pyvirtualdisplay for geckodriver to interact with
+        self.display = Display(visible=0, size=(1900, 1080))
```

Comment:
@hackdna I just chose a screen resolution that I envisioned would be about the norm of what most people use nowadays. If we make the resolution too small, pages wont render properly, and tests will probably fail.